crates/oxc_ast/src/ast/ts.rs

```diff
@@ -41,22 +41,10 @@ pub struct TSEnumDeclaration<'a> {
     #[cfg_attr(feature = "serde", serde(flatten))]
     pub span: Span,
     pub id: BindingIdentifier,
-    #[cfg_attr(feature = "serde", serde(flatten))]
-    pub body: TSEnumBody<'a>,
+    pub members: Vec<'a, TSEnumMember<'a>>,
     /// Valid Modifiers: `const`, `export`, `declare`
     pub modifiers: Modifiers<'a>,
 }
-/// Enum Body
-///
-/// A scope must be created on the enum body so this abstraction exists
```

Comment:
Any updates?

Reply:
I inlined enter / leave scope into the visitor function for now.